demo_plugin/modelgauge/tests/demo_04_using_annotation_test.py

```diff
@@ -56,7 +56,7 @@ def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]
 
     def get_annotators(self) -> Mapping[str, Annotator]:
         # Request the `DemoYBadAnnotator` be run, and key its results with "badness" for later.
-        return {"badness": DemoYBadAnnotator()}
+        return {"badness": DemoYBadAnnotator("badness_annotator")}
```

Comment:
default uids would be nice. does uid still need to be the first parameter?

Reply:
It does unfortunately :( I agree though, default uids would be nice, we can think about it some more later on.